components/ExerciseDetails/index.js

```diff
@@ -2,40 +2,108 @@ import styled from "styled-components";
 
 export default function ExerciseDetails({ exercise }) {
   return (
-    <>
-      <figure>
+    <Container>
+      <ImageWrapper>
         <ExerciseImage src={exercise.imageUrl} />
+      </ImageWrapper>
+      <TitleAndMuscleWrapper>
+        <ExerciseTitle>{exercise.name}</ExerciseTitle>
         <ExerciseDetailsMuscle>
-          {exercise.muscleGroups.join(" ")}
+          {exercise.muscleGroups.map((muscle, index) => (
+            <MuscleGroup key={index}>{muscle}</MuscleGroup>
+          ))}
         </ExerciseDetailsMuscle>
-      </figure>
+      </TitleAndMuscleWrapper>
       <ExerciseDetailsInstruction>
         {exercise.instructions.map((instruction, index) => (
-          <li key={index}>{instruction}</li>
+          <InstructionItem key={index}>
+            <InstructionNumber>{index + 1}</InstructionNumber>
+            {instruction}
+          </InstructionItem>
         ))}
       </ExerciseDetailsInstruction>
-    </>
+    </Container>
   );
 }
 
+const Container = styled.div`
+  font-family: Arial, sans-serif;
+  width: 100%;
+`;
+
+const ImageWrapper = styled.figure`
+  width: 100%;
+  height: 350px;
+  overflow: hidden;
+  margin: 0;
+  border-bottom-left-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+`;
+
 const ExerciseImage = styled.img`
-  width: 75vw;
-  height: auto;
-  border-radius: 1rem;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
 `;
+
+const TitleAndMuscleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 1rem;
+`;
+
+const ExerciseTitle = styled.h1`
+  margin: 0;
+  font-size: 1.5rem;
+  flex-shrink: 0;
+  color: #4D4020;
+`;
+
 const ExerciseDetailsMuscle = styled.figcaption`
```

Comment:
A figcaption must be a child of a figure element to make sure where it belongs to. I think that is not the best element for your purpose. You should use the next Image component for the picture and a div or something else for the muscles.